src/averageAges.js

```diff
@@ -15,11 +21,12 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredManArray = people.filter(person => century
+    ? (person.sex === 'm') && (Math.ceil(person.died / 100) === century)
```

Comment:
```suggestion
    ? (person.sex === 'm' && Math.ceil(person.died / 100) === century)
```

Reply:
Why we need outer brackets then?

Reply:
The same questions as[ here](https://github.com/mate-academy/js_average-ages/pull/1673#discussion_r1114559542)

Reply:
Can be omitted too. But we're using parenthesis for wrapping some meaningful blocks. It's easier to find the start and end.